modules/pytket-qiskit/pytket/extensions/qiskit/backends/aer.py

```diff
@@ -354,6 +370,7 @@ class AerBackend(_AerBaseBackend):
     _supports_shots = True
     _supports_counts = True
     _supports_expectation = True
+    _expectation_allows_nonhermitian = False
```

Comment:
I guess this is not supposed to be set for every backend? maybe a comment to say this is a variable just for this backend.

Reply:
Currently the default is `True`, so if non-hermitian isn't supported we have to set it to `False`. Maybe this is the wrong way round.

Reply:
I thought backends by default didn't support expectation values at all?

Reply:
Right, but this variable is only meaningful for those that do.

Reply:
I find the following in `backend.py`
```
   _supports_expectation = False
    _expectation_allows_nonhermitian = True
```
to be confusing, if we can flip the default I think it is less likely to lead to misunderstanding.

Reply:
Yeah I agree. Can we make a new task to change that (since it will require changes in both tket and extensions)?

Reply:
Or I could make all expectation-supporting backends set this explicity in this PR, and later we can change the default?

Reply:
I think doing them together with a new task is fine.